src/translator/response.h

```diff
@@ -77,6 +77,8 @@ struct Response {
 
 std::vector<Alignment> remapAlignments(const Response &first, const Response &second);
 
+std::vector<ByteRange> getWordByteRanges(Response const &response, size_t sentenceIdx);
```

Comment:
There are no plaintext clients to QE after outbound translation out of scope. Delivery for HTML is via HTML attributes. Can remove this?

Reply:
I'm in favour.

Reply:
So, we can just remove it. Right?

Reply:
It is being used by `common-impl.h` to extract the words for the QE test cases. @jerinphilip can I leave it up to you what you want to do with it? Either move it in there, or keep it in response.h. It's not publicly exposed.

Reply:
I can take care of this eventually as part of https://github.com/browsermt/bergamot-translator/issues/350. Marking as resolved.